samples/authorization/certificate.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: selfsigned
+  namespace: authorization
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: karavi-auth
+  namespace: authorization
+spec:
+  secretName: karavi-auth-tls
+  duration: 2160h # 90d
+  renewBefore: 360h # 15d
+  subject:
+    organizations:
+    - dellemc
+  isCA: false
+  privateKey:
+    algorithm: RSA
+    encoding: PKCS1
+    size: 2048
+  usages:
+    - server auth
+    - client auth
+  dnsNames:
```

Comment:
Is this sample certificate something customers would apply themselves? The dnsNames are hardcoded here but they need to include custom hostnames if they are specified. For example, if the customer puts `csm-authorization-proxy-server` for the the hostname, that name needs to be here too so the certificate works.

Reply:
Yes, this will be applied manually. I think its better to provide default values here to align with the proxy server CRD and then provide more detail on the configurable values in the documentation.